backend/deployment/values.yaml

```
@@ -0,0 +1,53 @@
+
+affinity: {}
+
+# Application configuration
+config:
+  # PostgreSQL Database Configuration
+  database:
+    host: "timescale-service"
+    port: 5432
+    username: "postgres"
+    database: "timescale"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)